libshaderc_util/include/libshaderc_util/compiler.h

```diff
@@ -111,7 +121,7 @@ class Compiler {
         suppress_warnings_(false),
```

Comment:
Maybe the constructor blurb should say it will default the target env to Vulkan.